c/alloc.c

```diff
@@ -843,7 +867,9 @@ char *safeMalloc64v2(unsigned long long size, int zeroOut, char *site,
       memset(data, 0, size);
     }
   } else{
-    printf("MALLOC failed, got NULL for size %llu at site %s\n",size, site);
+    if (MALLOC_TRACE_LEVEL >= 1){
+      printf("MALLOC failed, got NULL for size %llu at site %s\n",size, site);
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.